pkg/workloads/cortex/client/cortex/client.py

```diff
@@ -68,9 +68,9 @@ def deploy(
             deployment_name (string): deployment name
             api_name (string): API name
             model_path (string): S3 path to an exported model
+            model_format (string): model format, must be "tensorflow" or "onnx" (default: "onnx" if model path ends with .onnx, "tensorflow" if model path ends with .zip or is a directory)
```

Comment:
do we still default based on the path? If not we could add the defaulting code here, or remove the default note